cpp/clang-format-12

```diff
@@ -0,0 +1,93 @@
+---
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: true
```

Comment:
Just for clarity- is this same file as previous version or any modifications needed? and tested locally?

Reply:
copy of clang-format-11, tested on nmodl

Reply:
Ok